apis/model/v1alpha1/modelversion_types.go

```diff
@@ -45,9 +45,14 @@ type ModelVersionSpec struct {
 	Storage *Storage `json:"storage,omitempty"`
 
 	// ImageRepo is the image repository to push the generated model image. e.g. docker hub.  "modelhub/mymodel"
-	// A particular image tag is automatically generated by the system. e.g.  "modelhub/mymodel:v9epgk"
+	// A particular image tag is automatically generated by the system when ImageTag is nil, e.g.
+	// "modelhub/mymodel:v9epgk"
 	// +required
 	ImageRepo string `json:"imageRepo,omitempty"`
+
+	// ImageTag is the image tag suffixed with image repo, which indicates a specific image version/variant.
+	// +optional
+	ImageTag *string `json:"imageTag,omitempty"`
```

Comment:
does this need to be pointer ?

Reply:
we are able to distinguish whether `imageTag` has been set or not from `imageTag == nil`

Reply:
but I don't see where this differentiation is used ?

Reply:
https://github.com/kubedl-io/kubedl/pull/190/files#diff-7ad225ab0baaf20c14b612bf838e7c108a6ee66551c9016b6596ee6e22222526R141

Reply:
https://github.com/kubedl-io/kubedl/pull/190/files#diff-7ad225ab0baaf20c14b612bf838e7c108a6ee66551c9016b6596ee6e22222526R141